docs/components/compact.md

```diff
@@ -47,6 +47,8 @@ There's also a case when you might want to disable downsampling at all with `deb
 
 Ideally, you will have equal retention set (or no retention at all) to all resolutions which allow both "zoom in" capabilities as well as performant long ranges queries. Since object storages are usually quite cheap, storage size might not matter that much, unless your goal with thanos is somewhat very specific and you know exactly what you're doing.
 
+Setting no retention, ie `--retention.resolution-something=0d`, will mean samples at the `something` resolution level will be kept forever.
```

Comment:
```suggestion
Setting no retention, i.e. `--retention.resolution-X=0d`, will mean that samples at the `X` resolution level will be kept forever.
```
Mostly about the missing `.` after `i` and `e` :smile: WDYT?

Reply:
`resolution-X` is way clearer than `resolution-something` - thanks! I'll make this change :)